packages/gatsby-remark-prismjs/src/highlight-code.js

```diff
@@ -3,6 +3,7 @@ const _ = require(`lodash`)
 
 const loadPrismLanguage = require(`./load-prism-language`)
 const handleDirectives = require(`./directives`)
+const notSupportedLanguages = []
```

Comment:
How about we use a Set here?

```suggestion
const unsupportedLanguages = new Set()
```

Reply:
Nice catch!